CREDITS.json

```diff
@@ -0,0 +1,559 @@
+            "link": "https://www.nuget.org/packages/Microsoft.VisualStudio.Validation/15.3.32",
+            "version": "15.3.32",
+            "licenses": [
+                {
+                    "type": "MIT",
+                    "link": "https://raw.githubusercontent.com/Microsoft/vs-validation/653b14c15e/LICENSE",
+                    "name": "LICENSE"
+                }
+            ]
+        },
+        {
+            "name": "Microsoft.Xaml.Behaviors.Wpf",
+            "link": "https://github.com/microsoft/XamlBehaviorsWpf",
+            "version": "1.1.19",
+            "licenses": [
+                {
+                    "type": "MIT",
+                    "link": "https://github.com/microsoft/XamlBehaviorsWpf/blob/b88a49dfadcc98f41634624124ee19f9d9dc4df2/LICENSE",
+                    "name": "LICENSE"
+                }
+            ]
+        },
+        {
+            "name": "SkiaSharp",
+            "link": "https://github.com/mono/SkiaSharp",
+            "version": "1.68.3",
+            "licenses": [
+                {
+                    "type": "MIT",
+                    "link": "https://github.com/mono/SkiaSharp/blob/fd9484a06ae96f0195a80d93e80d6b54323f450a/LICENSE.txt",
+                    "name": "LICENSE.txt"
+                }
+            ]
+        },
+        {
+            "name": "SkiaSharp.HarfBuzz",
+            "link": "https://github.com/mono/SkiaSharp/tree/main/changelogs/SkiaSharp.HarfBuzz",
+            "version": "1.68.3",
+            "licenses": [
+                {
+                    "type": "MIT",
+                    "link": "https://github.com/mono/SkiaSharp/blob/fd9484a06ae96f0195a80d93e80d6b54323f450a/LICENSE.txt",
+                    "name": "LICENSE.txt"
+                }
+            ]
+        },
+        {
+            "name": "Splat",
+            "link": "https://github.com/reactiveui/splat",
+            "version": "2.0.0",
+            "licenses": [
+                {
+                    "type": "MIT",
+                    "link": "https://github.com/reactiveui/splat/blob/5a4912c1e0f5eeb434ec9f2452f6571d2e34c34b/LICENSE",
+                    "name": "LICENSE"
+                }
+            ]
+        },
+        {
+            "name": "Svg.Skia",
+            "link": "https://github.com/wieslawsoltes/Svg.Skia",
+            "version": "0.3.0",
+            "licenses": [
+                {
+                    "type": "MIT",
+                    "link": "https://github.com/wieslawsoltes/Svg.Skia/blob/c65367905b814451a9e3d00dc8da31ecb5d5914a/LICENSE.TXT",
+                    "name": "LICENSE.txt"
+                }
+            ]
+        },
+        {
+            "name": "System.Buffers",
+            "link": "https://www.nuget.org/packages/System.Buffers/4.5.1",
+            "version": "4.5.1",
+            "licenses": [
+                {
+                    "type": "MIT",
+                    "link": "https://github.com/dotnet/corefx/blob/master/LICENSE.TXT",
```

Comment:
This one and the next few don't have hashes on them.  Is that because the tag couldn't be found?

Reply:
This one in particular, the GitHub repo that is provided off of `https://www.nuget.org/packages/System.Buffers/4.5.1` does not contain a tag for `4.5.1`. The link here is what is provided by nuget.org.

I will double check the other ones.